README.asciidoc

```diff
@@ -0,0 +1,34 @@
+== Setup
+Install python 3.9 
+
+Install pip 3.9 and distutils for python 3.9:
+
+Use Pip to install the dependencis:
+
+pip3.9 install opencv-python pyrealsense2 numpy websocket-client pyserial

```

Comment:
Such lines can be formated to look like code blocks using the \`-symbols. Example `sudo rm -rf /`